tock/tock/templates/_navigation.html

```diff
@@ -1,51 +1,54 @@
-<nav role="navigation" class="usa-nav">
+<nav class="usa-nav" aria-label="Primary navigation">
   <div class="usa-nav__inner">
     <button class="usa-nav__close">
       {% load static %}
       <img src="{% static 'vendor/uswds_v2.3.1/img/close.svg' %}" alt="close">
     </button>
     <ul class="usa-nav__primary usa-accordion">
       <li class="usa-nav__primary-item">
-        <a href="{% url 'ProjectListView' %}" class="usa-nav__link">
-          <span>Projects</span>
+        <a href="/" accesskey="1" title="Home" aria-label="Home">Tock your time</a>
+      </li>
+      <li class="usa-nav__primary-item">
+        <a class="usa-nav__link" href="{{ x_tock_change_request_form_url }}" target="_blank">
+          <span class="usa-link--external">Request changes to your time</span>
         </a>
       </li>
       {% if request.user.is_staff %}
       <li class="usa-nav__primary-item">
         <button class="usa-accordion__button usa-nav__link" aria-expanded="false" aria-controls="reports-nav">
-          <span>Reports</span>
+          <span>Review summaries</span>
```

Comment:
Does "review summaries" make more sense than "reports" for the items in the tab? I'm wondering if it's an artifact from the mock that's changed in the interim.

Reply:
I'm not sure, tbh. I think it was an attempt to make all of the nav items begin with verbs, but in that case we could just say "View reports" and leave it at that?

Reply:
I pulled this up locally and poked around a bit. I'd like to propose this verbiage for the nav:

* Tock ( or Tock time, or Tock your time. No strong preference here.)
* Request changes 
* View reports
* View projects
* View users

Reply:
![okay](https://media.giphy.com/media/VbP5CZlMuWm7HPKrKf/giphy.gif)